lib/class-wp-rest-swagger-controller.php

```diff
@@ -248,9 +248,16 @@ function ($matches) use (&$defaultidParams){
 						if(!empty($pdetails['type'])){
 							if($pdetails['type']=='array'){
 								$parameter['type']=$pdetails['type'];
-								$parameter['items']=array('type'=>'string');
+								$parameter['items']=array(
```

Comment:
I had a previous request (that I should have merged) that had the following:
```
if($pdetails['items']) {
	$parameter['items']=$pdetails['items'];
} else {
        $parameter['items']=array('type'=>'string');
}
```
This was to cope with situations where $pdetails['items'] did not exist.

Reply:
Good catch. Added these guards in both places.